src/content/docs/pt-br/core-concepts/astro-syntax.mdx

```diff
@@ -0,0 +1,178 @@
+---
+title: Sintaxe do Astro
+description: Uma introdução à sintaxe de componente .astro .
+i18nReady: true
+---
+
+**Se você conhece HTML, você já sabe o suficiente para escrever seu primeiro componente Astro.**
+
+A sintaxe de componente Astro é um superconjunto do HTML. A sintaxe foi [projetada para sentir familiar a qualquer um com experiência em escrever HTML ou JSX](#diferenças-entre-astro-e-jsx) e adiciona suporte para incluir componentes e expressões do JavaScript.
+
+
+## Expressões parecidas com JSX
+
+Você pode definir variáveis do JavaScript locais dentro do script do frontmatter do componente entre as duas cercas de código (`---`) de um componente Astro. você pode então injetar essas variáveis no template HTML do componente usando expressões parecidas com JSX.
+
+:::note[dinâmico vs reativo]
+Usando essa abordagem, você pode incluir valores **dinâmicos** que são calculados no frontmatter. Mas uma vez incluídos, esses valores não são **reativos** e nunca mudarão. Os componentes Astro são modelos que só executam uma vez, durante a etapa de renderização.
+
+Veja abaixo mais exemplos de [diferenças entre Astro e JSX](#diferenças-entre-astro-e-jsx).
+:::
+
+### Variáveis
+
+Variáveis locais podem ser adicionadas ao HTML usando a sintaxe de chaves:
+
+```astro title="src/components/Variaveis.astro" "{nome}"
+---
+const nome = "Astro";
+---
+<div>
+  <h1>Olá {nome}!</h1>  <!-- Produz <h1>Olá Astro!</h1> -->
+</div>
+```
+
+### Atributos Dinâmicos
+
+Variáveis locais podem ser usadas em chaves para passar valores de atributos tanto para elementos HTML como para componentes:
```

Comment:
Nitpick: o "in" do original se refere principalmente a estar no meio das chaves, eu mudaria isso para ficar um pouco mais claro.
```suggestion
Variáveis locais podem ser usadas entre chaves para passar valores de atributos tanto para elementos HTML como para componentes:
```